### PR TITLE
Migrate slack-infra images

### PR DESCRIPTION
**Pull request by ameukam**
Signed-off-by: Arnaud Meukam <ameukam@gmail.com>

### CODE REVIEW

**Comment by ameukam** [comment]
Ref: https://github.com/kubernetes/k8s.io/issues/751

/assign @bartsmykla @spiffxp

File: config/jobs/kubernetes/test-infra/test-infra-trusted.yaml [hunk 1]
```diff
@@ -761,20 +761,20 @@ postsubmits:
...
       testgrid-num-failures-to-alert: "1"
-      testgrid-alert-email: ktbry@google.com, k8s-infra-oncall@google.com
+      testgrid-alert-email: ktbry@google.com, k8s-infra-oncall@google.com, k8s-infra-staging-slack-infra@kubernetes.io
       testgrid-dashboards: sig-contribex-slack-infra
     decorate: true
     branches:
     - ^master$
     spec:
-      serviceAccountName: deployer # TODO(fejta): should be pusher
+      serviceAccountName: pusher
```
Comment (spiffxp) [inline config/jobs/kubernetes/test-infra/test-infra-trusted.yaml:770]: this should be gcb-builder

Comment (ameukam) [reply config/jobs/kubernetes/test-infra/test-infra-trusted.yaml]: Fixed.

**Comment by spiffxp** [comment]
ahhhh sorry, I didn't notice which file this was in

can you move this job out into its own file in `config/jobs/image-pushing` and update it to use `cluster: k8s-infra-prow-build-trusted`?

**Comment by ameukam** [comment]
@spiffxp PR updated.

**Pull request merged**